esrally/utils/io.py

```diff
@@ -375,7 +375,7 @@ def guess_install_location(binary_name, fallback=None):
     """
     Checks whether a given binary is available on the user's path.
 
-    :param binary_name: The name of the binary, e.g. tail, gradle, mvn.
```

Comment:
Well, the comment is not wrong... :)

Reply:
Yes, the idea here is that I'd dissuade future developers from doing any gradle guesswork (even though the function of course can still be used for anything in `PATH`). Will revert.